README.md

```diff
@@ -471,6 +469,17 @@ Currently the available configuration properties are:
 [format] output format = TEXT
 ```
 
+#### Cromwell
+
+This command is creating an auto generation cromwell.conf in workspace.
+
+```
+Usage: terra cromwell [COMMAND]
+Generate a cromwell config in workspace ai notebook.
+Commands:
+  generate-config  Autogenerate a cromwell.conf under the user specified path.
+```
```

Comment:
Please also add that user needs to update the root to point to a gcs bucket of their choice.

### Cromwell with jupyter notebook
Cromwell and [cromshell](https://github.com/broadinstitute/cromshell) are installed on notebook under /home/jupyter/cromwell directory.  First, find your cromwell.conf under /home/jupyter/cromwell and update the gcs bucekt. 

Then start the cromwell server:

```
java -Dconfig.file=path/to/cromwell.conf -jar cromwell-81.jar server
```

Cromwell server will be running on localhost:8000.

Open another terminal window and type `cromshell`. On first time usage, it will prompt you to enter the port where cromwell server is running. Set the server to `localhost:8000` in cromshell. Then you can start running your workflow using cromshell in the notebook. e.g. ` cromshell submit workflow.wdl inputs.json options.json dependencies.zip`.  See https://github.com/broadinstitute/cromshell for details

something like this. You can rephrase it to make it clearer.